lms/djangoapps/teams/views.py

```diff
@@ -353,7 +355,10 @@ class TeamsListView(ExpandableFieldViewMixin, GenericAPIView):
     """
 
     # OAuth2Authentication must come first to return a 401 for unauthenticated users
```

Comment:
OAuth2Authentication will be updated to OAuth2AuthenticationAllowInactiveUser in this comment

Reply:
done